lib/katalyst/tables/backend.rb

```diff
@@ -17,9 +17,10 @@ module Backend
       # Sort the given collection by params[:sort], which is set when a user
       # interacts with a column header in a frontend table view.
       #
+      # @param sort [String] Sort ordering. Defaults to params[:sort]
```

Comment:
@sobakasu could you please explain the use case you have in mind here?

Reply:
use case is to clean up some code like this:

```
      params[:sort] ||= "published_at desc"
      @critic_reviews = sort_and_paginate(@form.reviews)
```

The intent is to have a default sort ordering.
This could be simplified to

```
      @critic_reviews = sort_and_paginate(@form.reviews, @form.sort)
```

with an search form attribute (scope "")

```
attribute :sort, :string, default: "published_at desc"
```